collectors/metrics/test/integration/setup.sh

```diff
@@ -2,181 +2,182 @@
-	echo -n "Apply Deployment yaml : " && kubectl apply -f ./metrics-collector/test/integration/manifests/thanos-api.yaml
-	echo -n "Apply Service yaml : " && kubectl apply -f ./metrics-collector/test/integration/manifests/thanos-service.yaml
-	echo "Waiting 2 minutes for observatorium and thanos to start... " && sleep 120
+  echo "=====Setting up thanos in kind cluster====="
+  echo -n "Apply create pvc yaml : " && kubectl apply -f ./metrics-collector/test/integration/manifests/thanos-pvc.yaml
+  echo -n "Apply configmap with hashring yaml : " && kubectl apply -f ./metrics-collector/test/integration/manifests/thanos-configmap.yaml
+  echo -n "Apply Deployment yaml : " && kubectl apply -f ./metrics-collector/test/integration/manifests/thanos-api.yaml
+  echo -n "Apply Service yaml : " && kubectl apply -f ./metrics-collector/test/integration/manifests/thanos-service.yaml
+  echo "Waiting 2 minutes for observatorium and thanos to start... " && sleep 120
 }
 
 deploy_prometheus_operator() {
-	echo "=====Setting up prometheus in kind cluster=====" 
-
-    WORKDIR=`pwd`
-    echo "Install prometheus operator." 
-    echo "Current directory"
-    echo $(pwd)
-    cd ${WORKDIR}/..
-    git clone https://github.com/coreos/kube-prometheus.git
-    echo "Replace namespace with openshift-monitoring"
-    $sed_command "s~namespace: monitoring~namespace: openshift-monitoring~g" kube-prometheus/manifests/*.yaml
-    $sed_command "s~namespace: monitoring~namespace: openshift-monitoring~g" kube-prometheus/manifests/setup/*.yaml
-    $sed_command "s~name: monitoring~name: openshift-monitoring~g" kube-prometheus/manifests/setup/*.yaml
-    $sed_command "s~replicas:.*$~replicas: 1~g" kube-prometheus/manifests/prometheus-prometheus.yaml
-    echo "Remove alertmanager and grafana to free up resource"
-    rm -rf kube-prometheus/manifests/alertmanager-*.yaml
-    rm -rf kube-prometheus/manifests/grafana-*.yaml
-    if [[ ! -z "$1" ]]; then
-        update_prometheus_remote_write $1
-    else
-        update_prometheus_remote_write
-    fi
-    
-    echo "HUB_KUBECONFIG" ${HUB_KUBECONFIG}
-    echo "KUBECONFIG" ${KUBECONFIG}
-    
-    echo "Creating prometheus manifests setup" && kubectl create -f kube-prometheus/manifests/setup
-    until kubectl get servicemonitors --all-namespaces; do date; sleep 1; echo ""; done
-    echo "Creating prometheus manifests" && kubectl create -f kube-prometheus/manifests/
-    rm -rf kube-prometheus
-    echo "Installed prometheus operator." 
-    sleep 60
-    echo -n "available services: " && kubectl get svc --all-namespaces
+  echo "=====Setting up prometheus in kind cluster====="
+
+  WORKDIR=$(pwd)
+  echo "Install prometheus operator."
+  echo "Current directory"
+  echo $(pwd)
+  cd ${WORKDIR}/..
+  git clone https://github.com/coreos/kube-prometheus.git
+  echo "Replace namespace with openshift-monitoring"
```

Comment:
im trying to understnad whats happening here? on KinD cluster, why do we need to use openshift-monitoring namespace with prom operator? 🤔